packages/api/cms-api/src/user-permissions/user-permissions.types.ts

```diff
@@ -0,0 +1,35 @@
+import { FindUsersArgs } from "./dto/paginated-user-list";
+import { User } from "./dto/user";
+import { UserPermission } from "./entities/user-permission.entity";
+import { ContentScope } from "./interfaces/content-scope.interface";
+import { Permission } from "./interfaces/user-permission.interface";
+
+export type PermissionConfiguration = Record<string, unknown>;
+
+export enum USERPERMISSIONS {
```

Comment:
Enums are pascal case, this would be UserPermissions

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/9272772317cfd1a27a08bcee65c3b59af5f4658c

Reply:
no reason for an enum?

(enums usually should be avoided)